dotcom-rendering/src/web/components/FetchOnwardsData.importable.tsx

```diff
@@ -32,7 +32,9 @@ export const FetchOnwardsData = ({
 	onwardsSource,
 	format,
 }: Props) => {
-	const { data, loading, error } = useApi<OnwardsResponse>(url);
+	const { data, loading, error } = useApi<OnwardsResponse>(url, {
+		errorRetryCount: 3,
```

Comment:
Is there a reason 3? [SWR uses an exponential backoff algorithm](https://swr.vercel.app/docs/error-handling#error-retry) which I would be wary of changing unless we had a specific reason.

Something to bare in mind is that it is not only server errors, but timeouts due to connections, network errors etc which can cause these, which is why the above algorithm is useful.